src/pages/Register/Register.js

```diff
@@ -1,7 +1,134 @@
+import {useEffect, useState} from "react";
+import {useDispatch, useSelector} from "react-redux";
+import {useNavigate} from "react-router-dom";
+import Box from "@mui/material/Box";
+import {Button, Link, Paper, Stack, TextField} from "@mui/material";
+import Typography from "@mui/material/Typography";
+import {register} from "store/auth/authActions";
+import {emailValid, passwordValid} from "utils/validators";
 import classes from "./Register.module.scss";
 
+const initialState = {
+    "name": {
+        "value": "",
+        "error":false
+    },
+    "surname": {
+        "value": "",
+        "error":false
+    },
+    "photo": {
+        "value": "",
+        "error":false
+    },
+    "email": {
+        "value": "",
+        "error":false
+    },
+    "username": {
+        "value": "",
+        "error":false
+    },
+    "password": {
+        "value": "",
+        "error":false
+    },
+    "passwordConfirmation": {
+        "value": "",
+        "error":false
+    }
+}
+const validate = (email, password, passwordConfirmation) => {
+    let error = "";
+    if (!emailValid(email)) {
+        error = "This is not a valid format of email.";
+    }
+    else if (!passwordValid(password)) {
+        error = "Password should contain numbers, uppercase and lowercase letters and special characters.";
+    }
+    else if (passwordConfirmation !== password) {
+        error = "Passwords should match.";
+    }
+    return error;
+}
+
 const Register = () => {
-    return <h1>Register page!</h1>
+    const [formState, setFormState] = useState(initialState);
+    const [registrationSubmit, setRegistrationSubmit] = useState(false);
```

Comment:
same comment here as in the login component, we don't need this state and to set updateField in registrationHandler. Please apply same changes as in login